clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -532,6 +532,9 @@ def err_stack_tagging_requires_hardware_feature : Error<
   "'-fsanitize=memtag-stack' requires hardware support (+memtag). For Armv8 or "
   "Armv9, try compiling with -march=armv8a+memtag or -march=armv9a+memtag">;
 
+def err_pauth_cpu_feature_missing : Error<
+  "A feature is requested that needs CPU with Pointer Authentication support">;
```

Comment:
Nit: according to llvm coding standards, error and warning messages should start with a lower-case letter https://llvm.org/docs/CodingStandards.html#error-and-warning-messages.

The message text looks OK, but I'll think about it a bit more - I believe it can be re-phrased in a better way (already good enough now though).

Reply:
Thanks! Updated the message.